src/fairseq2/models/hub.py

```diff
@@ -130,6 +356,32 @@ def load_model(
 
         return cast(ModelT, model)
 
+    @overload
+    def load_custom_model(
+        self,
+        path: Path,
+        config: ModelConfigT,
+        *,
+        gangs: Gangs | None = None,
+        dtype: DataType | None = None,
+        mmap: bool = False,
+        restrict: bool | None = None,
+        progress: bool = True,
+    ) -> ModelT: ...
+
+    @overload
+    def load_custom_model(
+        self,
+        path: Path,
+        config: ModelConfigT,
+        *,
+        device: Device | None = None,
+        dtype: DataType | None = None,
+        mmap: bool = False,
+        restrict: bool | None = None,
+        progress: bool = True,
+    ) -> ModelT: ...
+
     def load_custom_model(
```

Comment:
qq - does this support loading models checkpoints that we created during training, e.g. for a quick manual evaluation run with EvalRecipe without consolidating and creating a card for it?

If yes, this would be the better example to showcase/or at least mention, this is a common friction point I heard from MMS.

Reply:
I plan to write much more detailed documentation on assets. There are so many features that many people are not aware of. That is one of my goals for next week (along with model authoring)